app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,27 @@
+class CommentsController < ApplicationController
+  def new
+    @user = current_user
+    @post = Post.find(params[:id])
```

Comment:
- [ ] We dont have to need @user & @post here , so kindly remove it.

Reply:
- [ ] @biruk-bereka as previously requested kindly remove like 3 and 4 for @user and @post, in this new method we don't need them as we will define them in `create` a method where they are required for creation of a new comment